app/assets/stylesheets/components/_footer.scss

```diff
@@ -25,7 +26,7 @@
   opacity: 1;
 }
 .footer .fa-heart {
-  color: #D23333;
+  color: #d23333;
```

Comment:
same as above.
Prefer using SCSS variables like `$light-gray` for consistency around the app and reusability

Reply:
Thanks, Done!